Makefile

```diff
@@ -5,36 +5,28 @@ all:
 	@echo make restart
 	@echo make keyboard-dev
 
-install-audio-receiver:
-	wget -q https://github.com/nicokaiser/rpi-audio-receiver/archive/main.zip
-	unzip main.zip
-	rm main.zip
-	
-	cd rpi-audio-receiver-main;	sudo ./install.sh
-
-dependencies: install-audio-receiver
+dependencies:
 	sudo apt install curl virtualenv espeak mpc mpd
 	virtualenv -p /usr/bin/python3 virtualenv
 	./virtualenv/bin/pip install --upgrade -r requirements.txt
 	echo "Reboot is recommended now"
 
 install:
-	# Install the systemd service...
-	# Create systemd directories
-	mkdir -p ~/.config/systemd/user
-
-	# Copy the service file to systemd directories
-	## cp spotify-alarm-clock.service ~/.config/systemd/user
-	# Reload the systemd daemon: https://askubuntu.com/a/1143989/413683
-	## systemctl --user daemon-reload
-	# Enable the service to start system boot (I hope)
-	## systemctl --user enable spotify-alarm-clock
-
-	# copy the .service file to global systemd directory
-	cp spotify-alarm-clock.service /etc/systemd/service
+	# Install the systemd service
+	# For the program to run in the background and start on boot
+	sudo cp spotify-alarm-clock.service /etc/systemd/system
 	sudo systemctl daemon-reload
 	sudo systemctl enable spotify-alarm-clock
 
+aliases:
+	cat bash_aliases >> ~/.bash_aliases
+	source ~/.bashrc
+
+radio-stations:
+	mpc add https://stream-relay-geo.ntslive.net/stream
+	mpc add https://stream-relay-geo.ntslive.net/stream2
+	mpc repeat on
```

Comment:
Adds radio stations, which is necessary for radio/`mpc` to work, and was previously undocumented